www/icapeg/src/etc/rc.d/icapeg

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+# PROVIDE: icapeg
+# REQUIRE: SERVERS
+# KEYWORD: shutdown
+#
+
+. /etc/rc.subr
+
+name=icapeg
```

Comment:
I had added it here as I've found many plugins to do the same.

Among plugins that do this are:
1. `acme-client`
2. `crowdsec`
3. `maltrail`
4. `openconnect`
5. `stunnel`
6. `tinc`

Among many others/

Reply:
I did not say that you did something wrong - it looks like you did your best to work around a faulty package. In my opintion, this file should be placed in the FreeBSD upstream package for icapeg. I would recommend you to ask the upstream author if he/she can include that file into the FreeBSD base package. Then this file will not be part of a OPNsense UI plugin and they can change it when they update the service. Otherwise an upstream change may break the plugin.